platform/viewer/cypress/integration/pwa/OHIFExtensionVTK.spec.js

```diff
@@ -57,7 +56,54 @@ describe('OHIF VTK Extension', () => {
       .should('be.visible')
       .contains('Layout');
 
-    cy.wait(3000);
-    cy.percyCanvasSnapshot('VTK Extension');
+    // Visual comparison
+    cy.screenshot();
+    cy.percyCanvasSnapshot('VTK initial');
```

Comment:
I'm mixed on this one.
It's been working well so far, but I could see us accidentally grabbing a "9%" loaded vs. "10%" loaded, depending on networks speeds at time of testing. We may need to suspend this test until we can assert the viewports are loaded

Reply:
Unless we already do? I can't remember if we "wait for Loading..." or "reformatting..." to no longer be showing before snapping

Reply:
Yes, now we are waiting for "...% Reformatting" to be completed and no longer be displayed at the viewports.